app/experimenter/experiments/forms.py

```diff
@@ -689,6 +689,27 @@ class Meta:
             "qa_status",
         )
 
+    def clean(self):
+        cleaned_data = super().clean()
+        if (
+            "risk_technical" in cleaned_data
+            and "risk_technical_description" in cleaned_data
+        ):
+            # Both checked, now we need to do an invariance check on these
+            # two. This is to match what's done with jQuery in the form:
+            # the 'risk_technical_description' needs to be required
+            # if 'risk_technical' is truthy.
+            if cleaned_data.get("risk_technical") == "True":
```

Comment:
It's weird to me that these are strings like that. Even after supposed cleaning. I've never seen that before. Is there a better way to do this?

Reply:
Yeah that is weird?

Reply:
It looks like this is because the base form field is a `ChoiceField` which coerces all values to strings. Since the underlying model for these fields is a boolean we should use `TypedChoiceField` here and provide a proper `coerce` function to make this nicer.